packages/ui/src/Provider/Context.ts

```diff
@@ -1,10 +1,16 @@
-import { createContext } from 'react';
+import { createContext, ElementType } from 'react';
 import defaultTheme from './theme/defaultTheme';
 import { ThemeInterface } from './theme/theme.types';
 
+const getElement = (element: ElementType, props: { href?: string }): ElementType =>
+  props.href ? 'a' : element;
+
 export interface ContextInterface {
   theme: ThemeInterface;
   replaceTheme?: (nextTheme: ThemeInterface) => void;
+  getElement: (element: ElementType, props: { href?: string }) => ElementType;
```

Comment:
Destruct href

Reply:
тут desctuct лишний, это только определение типа, добавил интерфейс для этих свойств, что бы было очевидней

here desctuct is superfluous, it is only a type definition, added an interface for these properties, it would be more obvious